_docs/setup/kubernetes/ansible-install.md

```diff
@@ -101,12 +100,12 @@ ansible-playbook main.yml -e '{"cluster_flavour": "k8s", "cmd_path": "~/kubectl"
 
 - User wants to install Istio on Openshift with settings other than the default
 ```bash
-ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "jaeger": true, "delete_resources": true}}'
+ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "delete_resources": true}}'
```

Comment:
this document could probably overall use some rework around the versions given that we are on 0.7.0, but imo looks good enough for merge.

Reply:
I updated the example to use `0.6.0`